lib/features/menu/coffeeMenu/screen/coffee_menu_screen.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import '../widget/coffee_card_widget.dart';
+
+class CoffeeMenuScreen extends StatefulWidget {
+  const CoffeeMenuScreen({Key? key}) : super(key: key);
+
+  @override
+  State<CoffeeMenuScreen> createState() => _CoffeeMenuScreenState();
+}
+
+class _CoffeeMenuScreenState extends State<CoffeeMenuScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          "Coffee",
```

Comment:
I found a cool stuff about hardcoded strings. To be faster later, when you need to add localizations, you need to find hardcoded strings. To do it faster, you can create an extension
```dart
extension StringExtension on String {
    String get hardcoded => this;
}
```

So after that you can mark hardcoded strings as `'Coffee'.hardcoded`, to be able to find them later